testsuite/gnat2goto/lib/python/test_support.py

```diff
@@ -102,15 +101,13 @@ def prove(cbmcargs="", main="", debug=False):
     """Call gnat2goto (and cbmc) on all *.adb files from the current directory
 
     PARAMETERS
-      none: yet
+      cbmcargs: arguments to be passed to CBMC
+      main: name of the file containing the entry point
+      debug: flag indicating if debug output should be printed out
     """
     for file in ada_body_files():
-        gnat2goto_success = process_gnat2goto(debug, file, main != "" and file != main)
-        if gnat2goto_success == False:
+        gnat2goto_success = process_gnat2goto(debug, file, main == "" or file == main)
+        if not gnat2goto_success:
             return ""
     out = process_cbmc(debug, ada_body_files(), cbmcargs)
-    if debug:
```

Comment:
Shame on me, I didn't realise this was controlled through a flag. Maybe this shouldn't be removed after all, sorry

Reply:
Un-removed.